### DISCUSSION TITLE
Keywords - Search for bookmarks

### DISCUSSION THREAD

**Discussion by polinom11** [brookhong/Surfingkeys#1708] [Ideas]
Do you plan to support firefox bookmark keywords in the omnibar? 

![2022-03-14_12-41-34 (2)](https://user-images.githubusercontent.com/48948406/158150009-659278d6-470d-43b6-855a-d77df8657bcf.png)

**Comment by zxyqq** [comment]
+1
it will be appriciated if it can support fuzzy search for bookmark

**Comment by Fred-Vatin** [comment]
@polinom11 
The last time I checked, Firefox didn't allow extension or had an api to expose those keywords. I’m pretty sure it is still the case.

@zxyqq 
Fuzzy search would be great. Requests already exist here: #1673, #1571.
